test/cypress/integration/worldpay-3ds-flex-ddc/payment-with-session-id.cy.test.js

```diff
@@ -189,27 +172,132 @@ describe('Worldpay 3ds flex card payment flow', () => {
     })
   })
 
-  setUpAndCheckCardPaymentPage()
   describe('DDC times out', () => {
-    it('iframe post should still submit the form but without the worldpaySessionId', () => {
+    it('Should submit the form from the iframe post without the worldpaySessionId ', () => {
+      cy.log('Should setup a standard payment page')
+
+      cy.task('setupStubs', createPaymentChargeStubsEnglish)
+      cy.visit(`/secure/${tokenId}`)
+
+      // 1. Charge will be created using this id as a token (GET)
+      // 2. Token will be deleted (DELETE)
+      // 3. Charge will be fetched (GET)
+      // 4. Service related to charge will be fetched (GET)
+      // 5. Charge status will be updated (PUT)
+      // 6. Client will be redirected to /card_details/:chargeId (304)
+      cy.location('pathname').should('eq', `/card_details/${chargeId}`)
+
+      cy.task('clearStubs')
+      cy.task('setupStubs', checkCardDetailsStubs)
+
+      cy.server()
+      cy.route('POST', `/check_card/${chargeId}`).as('checkCard')
+
+      cy.log('Should enter and validate a correct card')
+
+      cy.get('#card-no').type(validPayment.cardNumber)
+      cy.get('#card-no').blur()
+
+      // 7. Charge status will be fetched (GET)
+      // 8. CardID POST to verify that entered card is correct - this is configured to return valid
+      cy.wait('@checkCard')
+      cy.get('#card-no').should('not.have.class', 'govuk-input--error')
+
+      cy.log('Should enter payment details')
+
+      cy.get('#expiry-month').type(validPayment.expiryMonth)
+      cy.get('#expiry-year').type(validPayment.expiryYear)
+      cy.get('#cardholder-name').type(validPayment.name)
+      cy.get('#cvc').type(validPayment.securityCode)
+      cy.get('#address-line-1').type(validPayment.addressLine1)
+      cy.get('#address-city').type(validPayment.city)
+      cy.get('#address-postcode').type(validPayment.postcode)
+      cy.get('#email').type(validPayment.email)
+
       const confirmPaymentDetailsStubsPop = _.cloneDeep(confirmPaymentDetailsStubs)
       confirmPaymentDetailsStubsPop.pop()
 
+      cy.task('clearStubs')
       cy.task('setupStubs', confirmPaymentDetailsStubs)
 
+      cy.log('Should submit the form from the iframe post but without the worldpaySessionId')
+
       cy.get('#card-details').submit().should($form => {
         const formVal = $form.first()[0].elements.worldpay3dsFlexDdcResult
         expect(formVal).to.eq(undefined)
       })
     })
   })
 
-  setUpAndCheckCardPaymentPageWithGooglePay()
-  describe('Web payment text is not displayed on DDC spinner screen', () => {
-    it('iframe post should still submit the form but without the worldpaySessionId', () => {
+  describe('DDC times out when Google pay is an option', () => {
+    it('Should submit the form from the iframe post without the worldpaySessionId when Google pay option is present', () => {
+      cy.log('Need setup a standard payment page before setting up the page with Google pay')
```

Comment:
Test would not work unless I set up a standard payment page first before setting up a payment page with Google pay.